.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,57 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.140.1/containers/dotnetcore
+{
+  "name": "C# (.NET 6)",
+  "build": {
+    "dockerfile": "Dockerfile",
+    "args": {
+      "VARIANT": "6.0",
+    }
+  },
+  "settings": {
+    "files.associations": {
+      "*.csproj": "msbuild",
+      "*.fsproj": "msbuild",
+      "*.globalconfig": "ini",
+      "*.manifest": "xml",
+      "*.nuspec": "xml",
+      "*.pkgdef": "ini",
+      "*.projitems": "msbuild",
+      "*.props": "msbuild",
+      "*.resx": "xml",
+      "*.rsp": "Powershell",
+      "*.ruleset": "xml",
+      "*.settings": "xml",
+      "*.shproj": "msbuild",
+      "*.slnf": "json",
+      "*.targets": "msbuild",
+      "*.vbproj": "msbuild",
+      "*.vsixmanifest": "xml",
+      "*.vstemplate": "xml",
+      "*.xlf": "xml",
+      "*.yml": "azure-pipelines"
+    },
+    // ms-dotnettools.csharp settings
```

Comment:
This is all duplicated with the .vscode/settings.json -- is there a way to avoid the duplication? And what is using this file?

Reply:
unfortunately there does not seem to be a way to do this today. The settings in this file are used by codespaces